modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -513,6 +513,7 @@ const FrontendDataSet = ({
 					activeDelta={paginationDelta}
 					activePage={pageNumber}
 					deltas={pagination?.deltas}
+					disableEllipsis={true}
```

Comment:
Hey @dsanz ,

Initially, I thought about something slightly similar and considering only performance, it makes total sense to me.

The reason why I didn’t send it like that was because of UX. I thought it might confusing for an end user to see how that button sometimes is enabled and sometimes is disabled (It’s unlikely an end user is going to figure out we disable it when the dropdown is too large for performance reasons), and I thought it might be prone to be reported as a perceived bug.

But if we only want to consider performance and not UX in this case, just confirm it to me here, and I’ll update it. 

Thanks!

Reply:
It makes sense to me. Let's apply your suggestion.

Thanks for your explanation, @dsanz . Very helpful!